app/models/embed/purl_xml_loader.rb

```diff
@@ -52,6 +53,10 @@ def stanford_only_unrestricted
       rights.stanford_only_unrestricted?
     end
 
+    def stanford_only_no_download
+      !public && !rights.stanford_only_downloadable?
+    end
+
     def controlled_digital_lending
```

Comment:
Public XML: A new restriction type we are catching (currently only for the PDF viewer, but could be elsewhere).